src/modules/rlm_python3/configure

```diff
@@ -1,51 +1,51 @@
 #! /bin/sh
 # From configure.ac Revision.
 # Guess values for system-dependent variables and create Makefiles.
-# Generated by GNU Autoconf 2.69.
-#
-#
-# Copyright (C) 1992-1996, 1998-2012 Free Software Foundation, Inc.
-#
+# Generated by GNU Autoconf 2.63.
```

Comment:
Can you update the `configure` using a version >= 2.69?

Reply:
Yep. Would any autoconf >= 2.69 work,or is there any flavour that you prefer (ie. the one with centos7, the one from debian 10....).